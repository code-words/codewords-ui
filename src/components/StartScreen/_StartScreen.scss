.StartScreen {
  position: relative;
  background: $archer-vice;
  background-position: center;
  background-size: cover;
  border: silver 3px solid;
  border-radius: 5px;
  padding: 30px;
  height: 60vh;
  width: 40vw;

  h2,h4 {
    font-family: $header-font;
    margin-bottom: 5px;
  }

  h4 {
    margin-bottom: 25px;
  }

  nav {
    height: 60%;
    margin: 10%;
    display: flex;
    flex-direction: column;
    align-items: flex-start;
  }

  .start-btn {
    height: 46px;
    width: 200px;
    margin: 15px;
  }

}

