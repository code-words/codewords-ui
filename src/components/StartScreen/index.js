import React from 'react';
import { NavLink } from 'react-router-dom';

const StartScreen = () => {
  return (
    <div className="backdrop">
      <section className="StartScreen">
        <article>
          <h2>Welcome to CodeWords</h2>
          <h4>We've got a mission for you agents!</h4>
          <p>Get your friends together to put your wits to the test. Play team vs team to see if you have what it takes to be the best damn operatives we've got. Give your partner one word hints to find the most case files.</p> 
        </article>
        <nav>
          <NavLink to="/new">START NEW GAME</NavLink>
          <NavLink to="/join">JOIN GAME</NavLink>
          <NavLink to="/rules">VIEW RULE LIST</NavLink>
        </nav>
      </section>
    </div>
  )
}

export default StartScreen;