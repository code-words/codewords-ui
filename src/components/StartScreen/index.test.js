import React from 'react';
import StartScreen from '.';
import { shallow } from 'enzyme';

describe('StartScreen', () => {
  let wrapper;
  let instance;

  beforeEach(() => {
    wrapper = shallow(<StartScreen />);
    instance = wrapper.instance();

  })

  it('should match component snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });

})