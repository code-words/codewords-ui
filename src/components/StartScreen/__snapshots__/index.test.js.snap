// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StartScreen should match component snapshot 1`] = `ShallowWrapper {}`;
